docs/csharp/language-reference/operators/delegate-operator.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "delegate operator - C# reference"
+ms.date: 07/18/2019
+helpviewer_keywords:
+  - "delegate [C#]"
+  - "anonymous method [C#]"
+---
+# delegate operator (C# reference)
+
+The `delegate` operator creates an anonymous method that can be converted to a delegate type:
+
+[!code-csharp-interactive[anonymous method](~/samples/csharp/language-reference/operators/DelegateOperator.cs#AnonymousMethod)]
+
+Beginning with C# 3, [lambda expressions](../../programming-guide/statements-expressions-operators/lambda-expressions.md) provide a more concise and expressive way to create an anonymous function. Use the [=> operator](lambda-operator.md) to construct a lambda expression:
```

Comment:
I like that idea.

This article should more clearly reference the discussion on capturing outer variables and closures in the article on [lambda expressions](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/statements-expressions-operators/lambda-expressions) in the programming guide. 

Consider adding a sentence specifically mentioning that discussion.

Reply:
@BillWagner please take a look at my attempt made in the latest commit. If it's fine with you, let's merge it, I'll update the programming guide in the next PR.